### PR TITLE
Finish reformatting of IntegralGrader, implement tests+docs

### PR DESCRIPTION
**Pull request by jolyonb**
This splits out the functionality in `IntegralGrader` that will be duplicated in `SumGrader` into a base class + subclass implementation. `SumGrader` should now be straightforward to implement on top of the base class.

This also implements new tests to get coverage back to 100%, and updates the documentation to include all new functionality.

### CODE REVIEW

**Comment by jolyonb** [comment]
Not sure why this isn't generating a message for coverage, but I'm pretty sure it's at 100%.

Ready for review (should be pretty straightforward).

**Comment by jolyonb** [comment]
Ooh, it actually worked! Wonders never cease! (and yes, 100%!)

File: mitxgraders/formulagrader/integralgrader.py [hunk 1]
```diff
@@ -86,7 +87,159 @@ def wrapper(x):
...
+class SummationGraderBase(AbstractGrader, MathMixin):
+    """
+    Abstract base class that incorporates the common components of IntegralGrader and SumGrader.
+    """
+    default_variables = merge_dicts(DEFAULT_VARIABLES, {'infty': float('inf')})
+
+    @abstractproperty
+    @property
+    def wording(self):
+        """Returns a dictionary with keys 'noun' and 'adjective' that describe the function of this class"""
```
Comment (ChristopherChudzicki) [inline mitxgraders/formulagrader/integralgrader.py]: 👍

**Review by ChristopherChudzicki — APPROVED** [review]

File: mitxgraders/formulagrader/integralgrader.py (hunk 1 above)
Comment (jolyonb) [reply mitxgraders/formulagrader/integralgrader.py]: Glad you enjoyed :-)

**Pull request merged**